git-hg/import-common.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 
 # import-common
-cd $WORKSPACE
+cd "$WORKSPACE" || exit 1
 export submodules="corba langtools jaxp jaxws hotspot nashorn jdk"
```

Comment:
Consider using explicit array rather than string parsing, e.g.
```
$ export submodules=(corba langtools jaxp jaxws hotspot nashorn jdk)
$ for i in ${submodules[@]}; do
> echo ${i}
> done
corba
langtools
jaxp
jaxws
hotspot
nashorn
jdk
$ 
```